src/cli/subcommand/install.rs

```diff
@@ -30,6 +31,9 @@ pub struct Install {
     )]
     pub no_confirm: bool,
 
+    #[clap(flatten)]
+    pub settings: CommonSettings,
```

Comment:
This allows the `./harmonic install` to pick up common settings, such as those set by env vars in the action.yml